stable/dotnetcore/src/Kubeless.WebAPI/Startup.cs

```diff
@@ -22,33 +22,38 @@ public Startup(IConfiguration configuration, IHostingEnvironment env)
 
         public void ConfigureServices(IServiceCollection services)
         {
-            services.AddMvc()
-                .SetCompatibilityVersion(CompatibilityVersion.Version_2_2);
+            services.AddControllers();
 
             var function = FunctionFactory.GetFunction(Configuration);
             var timeout = FunctionFactory.GetFunctionTimeout(Configuration);
-            var referencesPath = FunctionFactory.GetFunctionReferencesPath(Configuration);
 
-            services.AddSingleton<IInvoker>(new CompiledFunctionInvoker(function, timeout, referencesPath));
+            services.AddSingleton<IInvoker>(new CompiledFunctionInvoker(function, timeout, FunctionFactory.GetFunctionPublishPath()));
             services.AddSingleton<IParameterHandler>(new DefaultParameterHandler(Configuration));
         }
 
-        public void Configure(IApplicationBuilder app, IHostingEnvironment env)
+        public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
         {
             if (env.IsDevelopment())
             {
                 app.UseDeveloperExceptionPage();
             }
 
+            app.UseHttpsRedirection();

```

Comment:
I don't think we should enforce HTTPS redirection at Kubeless layer, this may be a problem in simpler scenarios which HTTPS is not a requirement.

Reply:
You're right, I'll remove it